pkg/istio/rbac/v2/provider.go

```diff
@@ -137,12 +151,21 @@ func (p *v2) ConvertAthenzModelIntoIstioRbac(athenzModel athenz.Model, serviceNa
 		if rule.To == nil {
 			continue
 		}
+
 		from_principal := &v1beta1.Rule_From{
 			Source: &v1beta1.Source{},
 		}
 		from_requestPrincipal := &v1beta1.Rule_From{
 			Source: &v1beta1.Source{},
 		}
+
+		// Used if role has opted for combination policies
+		from_principalAndRequestPrincipal := &v1beta1.Rule_From{
+			Source: &v1beta1.Source{},
+		}
+		from_principalAndNotRequestPrincipal := &v1beta1.Rule_From{
```

Comment:
I'm a little bit confused as to why we need this? If we have combination policies defined for an authz policy, do we not want it to be strictly be and AND statement of the proxy + token principal which is the from_principalAndRequestPrincipal?

Reply:
This has been discussed.